go/paddlecloud/pfsls.go

```diff
@@ -0,0 +1,131 @@
+package paddlecloud
```

Comment:
Can we merge content of these files(`pfscp.go`, `pfsls.go`, `pfsmkdir.go`, `pfsrm.go`) to `filemanager/pfsmodules/*.go`?

For example, in `ls.go`:
```go
// LsCmd means LsCommand structure.
type LsCmd struct {
	Method string
	R      bool
	Args   []string
}
// for subcommand
func (*LsCmd) Name() string { return "ls" }
func (*LsCommand) Synopsis() string { return "List files on PaddlePaddle Cloud" }
...
// original command implementations
func (p *LsCmd) ToURLParam() url.Values {
...
```

In `cmd/paddlecloud/paddlecloud.go`:
```go
subcommands.Register(&pfsmod.LsCmd{}, "")
```